runtime/compiler/control/CompilationRuntime.hpp

```diff
@@ -439,6 +439,9 @@ class CompilationInfo
          }
       }; // CompilationStatsPerInterval
 
+   static bool isJITServerEnabled() { return _isJITServerEnabled; }
```

Comment:
How are these methods going to be used? Why isn't enough the mode we set in persistentInfo?

Reply:
I thought we needed another runtime check. Will remove it.